### PR TITLE
feat: Update ComposeMessageView

### PR DESCRIPTION
**Pull request by valentinperignon**

### CODE REVIEW

File: Mail/Views/New Message/AutocompletionView.swift [hunk 1]
```diff
@@ -18,48 +18,78 @@
...
+                    if !isLastRecipient {
+                        IKDivider()
+                    }
                 }
             }
+        }
+        .onAppear {
+            updateAutocompletion(currentSearch)
+        }
+        .onChange(of: currentSearch, perform: updateAutocompletion)
```
Comment (adrien-coye) [inline Mail/Views/New Message/AutocompletionView.swift:57]: To have a cohesive experience with the search, should we debounce this as well ? What do you think ?

This is what we have in the search:
` .debounce(for: .seconds(0.3), scheduler: DispatchQueue.main)`

**Review by adrien-coye — APPROVED** [review]
I ran the code, It looks good honestly.

I have just one optional remark regarding debouncing, let me know what you think.
Thanks 👍

File: Mail/Views/New Message/AutocompletionView.swift (hunk 1 above)
Comment (valentinperignon) [reply Mail/Views/New Message/AutocompletionView.swift:57]: Seems a good idea, I will try it!

Comment (adrien-coye) [reply Mail/Views/New Message/AutocompletionView.swift]: Perfect, thanks for trying it out.

**Review by PhilippeWeidmann — APPROVED** [review]

**Pull request merged**